src/main/java/bio/terra/cli/service/AxonServerService.java

```diff
@@ -0,0 +1,205 @@
+package bio.terra.cli.service;
+
+import bio.terra.axonserver.api.GcpResourceApi;
+import bio.terra.axonserver.client.ApiClient;
+import bio.terra.axonserver.client.ApiException;
+import bio.terra.axonserver.model.ClusterMetadata;
+import bio.terra.axonserver.model.ClusterStatus;
+import bio.terra.cli.businessobject.Context;
+import bio.terra.cli.businessobject.Server;
+import bio.terra.cli.exception.SystemException;
+import bio.terra.cli.service.utils.HttpUtils;
+import com.google.auth.oauth2.AccessToken;
+import java.util.UUID;
+import javax.annotation.Nullable;
+import org.apache.http.HttpStatus;
+import org.apache.http.util.TextUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** Utility methods for calling Axon Server endpoints. */
+public class AxonServerService {
+  private static final Logger logger = LoggerFactory.getLogger(AxonServerService.class);
+  // the client object used for talking to Axon Server
+  private final ApiClient apiClient;
+
+  /**
+   * Constructor for class that talks to Axon Server. If the accessToken is null, only
+   * unauthenticated endpoints can be called.
+   */
+  private AxonServerService(@Nullable AccessToken accessToken, Server server) {
+    this.apiClient = new ApiClient();
+
+    this.apiClient.setBasePath(server.getAxonServerUri());
+    if (accessToken != null) {
+      this.apiClient.setAccessToken(accessToken.getTokenValue());
+    }
+  }
+
+  public static AxonServerService fromContext() {
+    return new AxonServerService(Context.requireUser().getTerraToken(), Context.getServer());
+  }
+
+  /**
+   * Call the Axon Server GET
+   * /api/workspaces/v1/{workspaceId}/resources/{resourceId}/gcp/dataproccluster/metadata:
+   *
+   * @param workspaceId the workspace that contains the resource
+   * @param resourceId the cluster resource id
```

Comment:
Minor: for sake of completeness - please add param (already added) as well as return in the comment docs

applies to all functions